ghost/admin/app/controllers/lexical-editor.js

```diff
@@ -1221,61 +1231,53 @@ export default class LexicalEditorController extends Controller {
         _timedSaveTask;
 
     /* Private methods -------------------------------------------------------*/
-
     _hasDirtyAttributes() {
         let post = this.post;
 
         if (!post) {
             return false;
         }
 
-        // if the Adapter failed to save the post isError will be true
-        // and we should consider the post still dirty.
+        // If the Adapter failed to save the post, isError will be true, and we should consider the post still dirty.
         if (post.get('isError')) {
             this._leaveModalReason = {reason: 'isError', context: post.errors.messages};
             return true;
         }
 
-        // post.tags is an array so hasDirtyAttributes doesn't pick up
-        // changes unless the array ref is changed
+        // Post tags comparison
```

Comment:
Can we switch back to the comments that explain _why_ this is necessary rather than a comment that just tells you what the code is doing?